### ISSUE TITLE
Allow account sign up without extensions

### ISSUE THREAD

**Issue by aaoconno** [microsoft/devhome#2473]
### Suggested new feature or improvement

Allow adding account before adding any extensions.

### Scenario

As a first time dev home user, I want to add an account, so I can use continue exploring adding widgets that I expect will need to be connected to my account. I get an warning saying I need to add a widget first, which seems to be the wrong way around.  

### Additional details

expect I can workaround issue adding widget first. then attach an account to it.

**Comment by krschau** [comment]
The answer to your suggestion might end up being "there's not a lot we can do without an account attached", but I want to be sure I understand the flow you're seeing and would prefer. Where do you see something saying you need to add a widget before you can connect an account? You can connect accounts at any time from the Settings > Accounts page.

**Comment by aaoconno** [comment]
Hello Kristen, Thanks for your reply. You are correct. it is extension not widget (as written in initial issue) that needs to be added first before an account can be added.  I agree with you too, that I will have to add an account at some stage to get an extension working. As a noob, I did go to Settings > Accounts > Add an account, thinking I'd need that first, and received the info message, 
"No Extensions found. Install an Dev Home extension that uses your developer account, then sign in to add the account [Find extensions] | [Cancel]".  Happy to close issue. I'll keep exploring. Really appreciate your response as this is the first time I've used Github to log suggestion. I hope I can be of more value next time.
